resolve-workgroup/decisions/AYD-RE-003-Color-look-publishing.md

```diff
@@ -0,0 +1,50 @@
+# AYD-RE-003 Colorspace and look publishing
+
+- **Status:** Proposed
+- **Impact:**  Medium
+- **Contributors:**:
+  - YNPUT Team <team@ynput.io>,
+  - Jakub Jezek <jakub@ynput.io>
+  - Tony Dorfmeister  (Bepic) @tweak-wtf
+  - Jiri Sindelar (Dazzle Pictures) @jrsndl
+- **Approved by:** -
+
+## Background
+
+Currently any grading work done in Resolve's Color Tab is not easily transferable into Nuke as a LUT. This is a problem because the color grading is an important part of the look of the final image. We need to find a way to transfer the color grading from Resolve to Nuke.
+
+## Options
+
+### Option 1
+
+Creating a LUT File from **Primary Color Grading** Nodes. The new effect type would improve existing workflows. This method takes cues from Hiero's soft effects publishing. Users can add the LUTs to their Nuke scripts either as Viewer Process Nodes or in a linkable form.
+
+#### Pros
+
+- Grading can be shared with compositors for review without having to embed it directly into the images.
+
+#### Cons
+
+- Not yet available Resolve API for this
```

Comment:
No matter the decision - it might be good to put in at least a formal request to Blackmagic Design about this.